health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/timelinePopUpWrapper.js

```
@@ -751,26 +43,27 @@ const TimelinePopUpWrapper = ({ onClose, businessId, heading }) => {
             }));
             setTimelineSteps(steps);
         }
-    }, []);
+    }, [workflowData]);
 
     return (
         <PopUp
             onClose={onClose}
             heading={t(heading)}
             onOverlayClick={onClose}
             children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)